src/phoenix/server/api/types/TimeSeries.py

```diff
@@ -70,17 +71,24 @@ def _get_timeseries_data(
 ) -> List[TimeSeriesDataPoint]:
     metric_cls = metric.value
     data = dimension[dataset_role.value]
-    if issubclass(metric_cls, NullaryOperator):
-        metric_instance = metric_cls()
-    else:
-        assert issubclass(metric_cls, UnaryOperator)
-        metric_instance = metric_cls(operand=Column(dimension.name))
-    if issubclass(metric_cls, DriftOperator):
+    metric_instance = metric_cls()
+    if isinstance(metric_instance, UnaryOperator):
+        metric_instance = replace(
+            metric_instance,
+            operand=Column(dimension.name),
+        )
+    if isinstance(metric_instance, DriftOperator):
```

Comment:
the fall-through from the above if statement is intended?

Reply:
yes, it's for `count` which is a `NullaryOperator` without arguments